destroy.py

```diff
@@ -0,0 +1,68 @@
+import subprocess
+import sys
+import os
+import re
+
+input('Empty the s3 bucket(s) manually, then press enter to continue.')
```

Comment:
I wonder if we could set a Terraform variable `force_destroy` or similar. Then if one or many resources need a force destroy to be applied in order to be destroyed, it can happen there.

The script would then perform the following ops

1. Add `force_destroy = true ` to terraform.tfvars
2. Run `terraform apply`
3. Run `terraform destroy`

That way if other resources in the future need similar treatment they can all react to that new variable.

What do you think? Worth doing now or should we keep that for a future version?

Reply:
Another nice solution is shown [here](https://gist.github.com/bassmanitram/53a57988d9f4e4a2ffc94aed789bbf9e). I will do a couple tests with it and let you know how it goes.

Reply:
After a couple tests I think the best solution is to is to apply with force_destroy first. It all runs fast enough to not be an issue and the other solution can't be applied to our current code. The issue is that as we are using a module, we are not directly controlling the resource, so we can't configure a provisioner on the resource to control it's destruction.